libs/langchain/langchain/vectorstores/azuresearch.py

```diff
@@ -74,20 +75,21 @@ def _get_search_client(
     scoring_profiles: Optional[List[ScoringProfile]] = None,
     default_scoring_profile: Optional[str] = None,
     default_fields: Optional[List[SearchField]] = None,
+    user_agent: Optional[str] = "langchain",
```

Comment:
Why make this overridable? Having a user agent helps when looking into issues for a particular library of app, variability doesn't help in this case. It might be useful for applications to add their own user agent string, but in that case I'd treat it as adding while keeping "langchain", not as replacing langchain.

Reply:
@pablocastro modified according to our chat